README.md

```diff
@@ -46,6 +49,9 @@ as compressing will transform the body.
 those listed below, [zlib](http://nodejs.org/api/zlib.html) options may be
 passed in to the options object.
 
+As for *brotli*, a default is set to compression level 4, unless
```

Comment:
Move this into the documentation section of the option that the user will use to change the value, rather than just at the top.

Reply:
The documentation is in Node's docs. There's no point in copying those options here...

Reply:
Can you at least show me an example for how to change the level through this module's interface? I cannot figure it out, at least.

Reply:
For example, is it not the existing [level option](https://github.com/expressjs/compression#level) of this module?